website/src/pages/api/log.ts

```diff
@@ -0,0 +1,26 @@
+import type { APIRoute } from 'astro';
+
+import { getInstanceLogger } from '../../logger.ts';
+import { type InstanceLogger, logMessageSchema } from '../../types/logMessage.ts';
+
+export const POST: APIRoute = async ({ request }) => {
+    const body = await request.json();
```

Comment:
Currently, everybody can access this endpoint. Should we restrict the access to only logged in users? This would prevent us from getting error messages from the organism pages. But in theory one can flood our endpoint his way?

Reply:
As discussed: 
* We want that every page can send log messages, also pages where users don't need to be logged in.
* There is no other proper protection, is it should be publicly available
* We only write log messages to the log.  We don't execute code or similar (assuming that winston doesn't have vulnerabilities). So we have a risk of being DDoSed / spamed, but otherwise we should be fine. DDoSing our server is quite easy anyway, because we have many LAPIS/SILO instances that do heavy computations.

We leave this endpoint open.